site/includes/contributors.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="row">
+	<h2 class="wb-inv">
+		{{#is page.language "fr"}}
+		Collaborateurs
+		{{else}}
+		Contributors
+		{{/is}}
+	</h2>
+	<section class="provisional gc-contributors col-md-12">
+		<div class="col-md-1">
+			<h3 class="provisional from">
+			{{#is page.language "fr"}}
+			De :
+			{{else}}
+			From:
+			{{/is}}
+			</h3>
+		</div>
+		<div class="col-md-11">
+			<ul>
+				<li><a href="#">{{#is page.language "fr"}}Page d'acceuil institutionnelles{{else}}Institutional landing page{{/is}}</a></li>
+				<li><a href="#">{{#is page.language "fr"}}[Ministère ou agence]{{else}}[Department or agency]{{/is}}</a></li>
+			</ul>
+		</div>
+	</section>
+</div>
```

Comment:
Keep only 1 section
Remove the first div and heading level 2.
change the h3 for h2 and add the CSS class "h3"
Remove the CSS "col-md-*"
```
<section>
<div class="row">
<div class="col-md-1">
titre
</div>
<div class="col-md-11">
List et lien des institution
</div>
</div>
</section>
```

Reply:
Ok. I've done this but now the two divs are not lining up vertically.